### DISCUSSION TITLE
Remove E2E tests that rely on Jest/Puppeteer infrastructure

### DISCUSSION THREAD

**Discussion by gigitux** [woocommerce/woocommerce#44946] [WooCommerce, Blocks, and Site Editing]
We should consider removing the E2E tests that currently rely on our old testing infrastructure. There are two primary reasons for this proposal:

### Flakiness and Continuous Integration failures:
The E2E tests using the old testing infrastructure have consistently proven to be flaky. They contribute to frequent CI failures, making it challenging to understand if the PR introduces some regressions.

### Compatibility with WordPress Versions:
Currently, these E2E tests are designed to run on WordPress 6.2. When WordPress 6.4 is released, our plugin no longer loads correctly on older versions like WordPress 6.2. Consequently, the E2E tests will fail due to this compatibility issue.

Given these issues, I’m proposing the following actions:

1. **Remove the E2E Tests that rely on E2E test infrastructure**
We should remove E2E tests that rely on the old testing infrastructure altogether. This will eliminate the ongoing issues with flakiness and CI failures associated with these tests.

2. **Identify critical E2E tests and migrate to the new infrastructure**
During the remotion of E2E tests, we should identify critical E2E tests - (e.g. Mini Cart tests) and migrate them to the new infrastructure.

Please share your thoughts, concerns, or suggestions regarding this proposal.

**Comment by opr** [comment]
The tests started totally failing today due to WC 8.3 being released.

I think we should try updating the WP version used in the tests. We can then check which tests fail and then skip the failing ones.

These ones should be moved to playwright as a priority.

**Comment by gigitux** [comment]
We wrapped up the migration. I'm going to close this discussion!
